app/src/main/java/org/oppia/android/app/options/AudioLanguageActivityPresenter.kt

```diff
@@ -12,7 +14,16 @@ class AudioLanguageActivityPresenter @Inject constructor(private val activity: A
   private lateinit var prefSummaryValue: String
 
   fun handleOnCreate(prefKey: String, prefValue: String) {
-    activity.setContentView(R.layout.audio_language_activity)
+    val binding = AudioLanguageActivityBinding.inflate(activity.layoutInflater, null, false)
```

Comment:
Instead of this, use `DataBindingUtil` to use `setContentView`

Reply:
Refactored this to use DataBindingUtil and pushed the code.

Thanks